src/main/scala/firrtl/Namespace.scala

```diff
@@ -84,5 +84,14 @@ object Namespace {
 
     namespace
   }
+
+  /** Initializes a [[Namespace]] for [[Module]] names in a [[Circuit]] */
```

Comment:
I'll need to check the Scaladoc (which is a mess right now anyway), but I believe that since Module and Circuit are in package firrtl.ir, but Namespace is just in package firrtl, this needs to be [[ir.Module]] and [[ir.Circuit]]